scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
I don't see any other option at the moment, will go ahead and implement the code to pull it from release page.